src/APIResourceManager.php

```diff
@@ -66,36 +61,31 @@ public function getRouteName($route)
     /**
      * Returns the versioned url.
      *
-     * @param  array|string  $name
-     * @param  mixed  $parameters
+     * @param string $name
+     * @param mixed $parameters
      */
     public function getRoute($name, $parameters = [], bool $absolute = true): string
     {
-        return route($this->getRouteName($name), $parameters, $absolute);
+        return route($this->getRouteName((string) $name), $parameters, $absolute);
     }
 
     /**
-     *  Get config considering the API name if present.
+     * Get config considering the API name if present.
+     *
+     * @param string $cfg Config path
+     * @param string $name Name of api if present
+     *
+     * @return mixed The result of the config
      */
-    protected function getConfig(string $key, string $apiName = null): ?string
+    protected function getConfig($cfg, $name = null)
```

Comment:
shall we make the param more expressive? like: `configPath`

Reply:
I'll be deleting this in the next PR